sympy/printing/str.py

```diff
@@ -402,7 +402,8 @@ def _print_FracElement(self, frac):
             return numer + "/" + denom
 
     def _print_Poly(self, expr):
-        terms, gens = [], [ self._print(s) for s in expr.gens ]
+        ATOM_PREC = PRECEDENCE["Atom"] - 1
```

Comment:
Redundant parentheses are syntactically valid, but for printing we should avoid them, because they make the expression harder to read and can confuse the reader as to why they are there.